docs/storage-schema.md

```diff
@@ -0,0 +1,61 @@
+## Metrics
+
+Heapster exports the following metrics to its backends. 
+
+| Metric Name        | Description                                                                                        | Type       | Units       | Supported Since |
+|--------------------|----------------------------------------------------------------------------------------------------|------------|-------------|-----------------|
+| uptime             | Number of milliseconds since the container was started                                             | Cumulative | Nanoseconds | v0.9            |
+| cpu/usage          | Cumulative CPU usage on all cores                                                                  | Cumulative | Bytes       | v0.9            |
+| memory/usage       | Total memory usage                                                                                 | Guage      | Bytes       | v0.9            |
+| memory/page_faults | Number of major page faults                                                                        | Guage      | Count       | v0.9            |
+| memory/working_set | Total working set usage. Working set is the memory being used and not easily dropped by the Kernel | Guage      | Bytes       | v0.9            |
+| network/rx         | Cumulative number of bytes received over the network                                               | Cumulative | Bytes       | v0.9            |
+| network/rx_errors  | Cumulative number of bytes received over the network                                               | Cumulative | Count       | v0.9            |
+| network/tx         | Cumulative number of bytes sent over the network                                                   | Cumulative | Bytes       | v0.9            |
+| network/tx_errors  | Cumulative number of errors while sending over the network                                         | Cumulative | Count       | v0.9            |
+
+*Note: Guage refers to instantaneous metrics*
+
+## Labels
+
+Heapster tags each metric with the following labels.
+
+| Label Name     | Description                                                                   | Supported Since | Kubernetes specific |
+|----------------|-------------------------------------------------------------------------------|-----------------|---------------------|
+| pod_id         | Unique ID of a Pod                                                            | v0.9            | yes                 |
+| container_name | User-provided name of the container or full cgroup name for system containers | v0.9            | No                  |
+| labels         | Comma-separated list of user-provided labels. Format is 'key:value'           | v0.9            | Yes                 |
```

Comment:
Ah yes, you're right I just checked.

On Tue, Mar 24, 2015 at 5:57 PM, Vish Kannan <notifications@github.com>
wrote:

> In docs/storage-schema.md
> <https://github.com/GoogleCloudPlatform/heapster/pull/189#discussion_r27087812>
> :
>
> > +| network/rx         | Cumulative number of bytes received over the network                                               | Cumulative | Bytes       | v0.9            |
> > +| network/rx_errors  | Cumulative number of bytes received over the network                                               | Cumulative | Count       | v0.9            |
> > +| network/tx         | Cumulative number of bytes sent over the network                                                   | Cumulative | Bytes       | v0.9            |
> > +| network/tx_errors  | Cumulative number of errors while sending over the network                                         | Cumulative | Count       | v0.9            |
> > +
> > +*Note: Guage refers to instantaneous metrics*
> > +
> > +## Labels
> > +
> > +Heapster tags each metric with the following labels.
> > +
> > +| Label Name     | Description                                                                   | Supported Since | Kubernetes specific |
> > +|----------------|-------------------------------------------------------------------------------|-----------------|---------------------|
> > +| pod_id         | Unique ID of a Pod                                                            | v0.9            | yes                 |
> > +| container_name | User-provided name of the container or full cgroup name for system containers | v0.9            | No                  |
> > +| labels         | Comma-separated list of user-provided labels. Format is 'key:value'           | v0.9            | Yes                 |
>
> I remember changing it to ':' some time back.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/heapster/pull/189/files#r27087812>
> .
>